package.json

```diff
@@ -51,8 +51,8 @@
     "lint": "eslint src --ext ts",
     "test": "node ./out/test/runTest.js",
     "test-coverage": "node ./out/test/runTest.js --coverage",
-    "prettier:write": "prettier --write \"src/**/*.{ts, js}\"",
-    "prettier:verify": "prettier --list-different \"src/**/*.{ts, js}\"",
+    "prettier:write": "prettier --write \"src/**/*.{ts, js}\" \"resources/instructions/**/*.html\"",
+    "prettier:verify": "prettier --list-different \"{src, resources/instructions}/**/*.{ts, js}\" \"resources/instructions/**/*.html\"",
```

Comment:
Nit: shouldn't the `--list-different` and `--write` arguments look basically the same? Specifically, should we pick one or the other of `\"src/**/*.{ts, js}\"` and `\"{src, resources/instructions}/**/*.{ts, js}\"`, for consistency?

(And I'd lobby for the former, just because it's easier to read.)